recipes-org/zynq-safemode-cfgs/zynq-safemode-cfgs.bb

```diff
@@ -0,0 +1,41 @@
+SUMMARY = "Packages containing older nilrd ARM .cfg files + a script to install them"
```

Comment:
How's this recipe used?

Reply:
I'm not actually sure. [Here's the history](https://github.com/ni/meta-orgconf/commits/nilrt/20.7/sumo/recipes-core/zynq-safemode-cfgs). I'm not confident enough with why it was added to assert that it should be removed.

Reply:
Hmmm. Does os-common build without it? I didn't see any codesearch references for this recipe.

> zynq-safemode-cfgs: add new recipe
This recipe creates safemode packages for restoring all nilrt supported
devices on the older OS version. The contents of each package is a *.cfg
file containing the necessary files for installation (itb, bitfiles,
preinst, postinst) and a standard script to install a cfg file.
These ipks will be downloaded on target from feeds during backwards
migration on arm devices based on their device code.

Is the package even added to our feeds? If the answers are no, and no, let's remove it?